javascript/javascript1/week3/homework-js1-w3.js

```diff
@@ -0,0 +1,120 @@
+//**********************************************/
+// WARM UP
+//**********************************************/
+
+//--------------------------
+// Item array removal
+//--------------------------
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1); // cannot use slice() here because slice doesn't modify the original array
+  }
+}
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//--------------------------
+// When will we be there??
+//--------------------------
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function time(obj) {
+  const timeInPercentage = obj.destinationDistance / obj.speed;
```

Comment:
We haven't learnt about it yet but since you have such a good grasp of Javascript you can look into [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). In particular destructuring of objects would be interesting here!

When reading the parameters of obj you could actually write it as so:
`({ speed, destinationDistance })`